schemas/2015-08-01/Microsoft.Cache.json

```diff
@@ -0,0 +1,160 @@
+                                    "oneOf": [
+                                        {
+                                            "enum": [
+                                                "Basic",
+                                                "Standard",
+                                                "Premium"
+                                            ]
+                                        },
+                                        {
+                                            "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+                                        }
+                                    ],
+                                    "description": "Microsoft.Cache/Redis/sku: determines features and pricing of the cache",
+                                    "default": "Standard"
+                                },
+                                "family": {
+                                    "oneOf": [
+                                        {
+                                            "enum": [
+                                                "C",
+                                                "P"
+                                            ]
+                                        },
+                                        {
+                                            "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+                                        }
+                                    ],
+                                    "description": "Microsoft.Cache/Redis/sku: along with capacity, determines capacity of the cache",
+                                    "default": "C"
+                                },
+                                "capacity": {
+                                    "oneOf": [
+                                        {
+                                            "type": "integer"
+                                        },
+                                        {
+                                            "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+                                        }
+                                    ],
+                                    "description": "Microsoft.Cache/Redis: along with family, determines capacity of the cache",
+                                    "default": 1
+                                }
+                            },
+                            "required": [
+                                "name",
+                                "family",
+                                "capacity"
+                            ]
+                        },
+                        "shardCount": {
+                            "oneOf": [
+                                {
+                                    "type": "integer"
```

Comment:
The JSON Schema Draft 4 specification defines a type called "positiveInteger" that might be useful here instead of "integer".